### ISSUE TITLE
The {questid} doesn't work in the quest display

### ISSUE THREAD

**Issue by doitliketyler** [LMBishop/Quests#436]
### Description of Bug

Tried setting up a quest using the locale placeholders combined with the {questid} variable/macro for the quests display item but it returns this error in chat:

"Attempt to reference unknown task 'questid'

I know these tags work in some places but not the quest itself.

### Steps to Reproduce

Create this quest and reload:

display:
  name: "Brew Potions" 
  lore-normal:
    - "Task: %quests_q:{questid}_p:task%"
    - "Progress: %quests_q:{questid}_p:progress%"
  type: "POTION"
placeholders:
  task: "Brew 5 Potions"
  progress: "{brew:progress}/5 Potions"
tasks:
  brew:
    type: "brewing"
    amount: 5   

### Environment

- Minecraft version: 1.19
- Server software: Paper
- Quests version: 3.13.1

### Agreements

- [X] I am running the latest version of Quests
- [X] I am certain this issue is unique and a similar issue is not currently open
- [X] I am using a version of Minecraft which is supported by Quests

**Comment by LMBishop** [comment]
I think a better solution would be an way to get those local placeholders without using PlaceholderAPI

**Comment by doitliketyler** [comment]
I completely agree and it would make editing quests a breeze without having to repeat yourself.
